src/App.tsx

```diff
@@ -1,10 +1,53 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Header } from './Components/Header';
+import { TodoList } from './Components/Todolist';
+import { Footer } from './Components/Footer';
+import { Notification } from './Components/Notification';
 
-const USER_ID = 0;
+const USER_ID = 6342;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [selectedStatus, setSelectedStatus] = useState('all');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [hasErrorFromServer, setHasErrorFromServer] = useState(false);
```

Comment:
you can use only one error state and check by using the 
```suggestion
  const [errorMessage, setErrorMessage] = useState('');
  const hasErrorFromServer = !!errorMessage || errorMessage !== '';
```